CODEOWNERS

```diff
@@ -45,13 +45,6 @@ src/cpp/ @kroening @tautschnig @peterschrammel
 
 # These files change frequently and changes are low-risk
 
-src/util/irep_ids.def @diffblue/cbmc-developers
```

Comment:
The comment above this line must be removed as well. It does _not_ refer to the bottom three lines that remain.

Reply:
Fixed